### DISCUSSION TITLE
Local Computer Search

### DISCUSSION THREAD

**Discussion by vasnt** [nilsherzig/LLocalSearch#42] [Q&A]
Can this tool search thru all folder & files in hard disk? Or other AI tools exist for it?

**Comment by nilsherzig** [comment]
no but give me a minute, I'm going to build a quick prototype

**Reply by nilsherzig** [reply]
Okay, I think it should be fairly doable to implement this (at least allowing it to index one or more folders), and I'm also very interested in doing so. But It's going to take a lot of time to make it work with different file formats. 

Im sure there are other projects out there which already do that better :)

**Reply by vasnt** [reply]
thanks for quick reply, i havent came across any AI based search tools yet. Even windows-microsoft has not yet implemented yet.
Recoll/everything (Non-AI) search tools exists.
